pkg/processor/crd/crd.go

```diff
@@ -21,11 +21,6 @@ const crdTeml = `apiVersion: apiextensions.k8s.io/v1
 kind: CustomResourceDefinition
 metadata:
   name: %[1]s
-  annotations:
-    cert-manager.io/inject-ca-from: {{ .Release.Namespace }}/{{ include "%[2]s.fullname" . }}-%[4]s
```

Comment:
Sorry, but we cannot remove cert-manager annotation. It is used for webhook validation in several projects.

Reply:
another optional? -crd-full-metadata

Reply:
I suggest to check if source CRD contains `cert-manager.io/inject-ca-from` annotation and remove it only if not.